packages/wonder-blocks-modal/src/components/__docs__/modal-panel.stories.js

```diff
@@ -0,0 +1,434 @@
+    docs: {
+        storyDescription: `Here is a dark \`<ModalPanel>\` with a header
+            and a footer. The \`<Button>\` in the footer must have the
+            \`light\` prop set to true in order to be visible on the dark
+            background.`,
+    },
+};
+
+export const TwoPanels: StoryComponentType = () => {
+    const mobile = "@media (max-width: 1023px)";
+    const desktop = "@media (min-width: 1024px)";
+
+    const twoPaneDialogStyle = {
+        [desktop]: {
+            width: "86.72%",
+            maxWidth: 888,
+            height: "60.42%",
+            minHeight: 308,
+        },
+        [mobile]: {
+            width: "100%",
+            height: "100%",
+            overflow: "hidden",
+        },
+    };
+
+    const panelGroupStyle = {
+        flex: 1,
+
+        [desktop]: {
+            flexDirection: "row",
+        },
+        [mobile]: {
+            flexDirection: "column",
+        },
+    };
+
+    return (
+        <View style={styles.previewSizer}>
+            <View style={styles.modalPositioner}>
+                <ModalDialog style={twoPaneDialogStyle}>
+                    <View style={panelGroupStyle}>
+                        <ModalPanel
+                            content={
+                                <View>
+                                    <Title>Sidebar</Title>
+                                    <Strut size={Spacing.large_24} />
+                                    <Body>
+                                        Lorem ipsum dolor sit amet, consectetur
+                                        adipiscing elit, sed do eiusmod tempor
+                                        incididunt ut labore et dolore magna
+                                        aliqua. Ut enim ad minim veniam, quis
+                                        nostrud exercitation ullamco laboris.
+                                    </Body>
+                                </View>
+                            }
+                            light={false}
+                            closeButtonVisible={false}
+                        />
+                        <ModalPanel
+                            content={
+                                <View>
+                                    <Title>Contents</Title>
+                                    <Strut size={Spacing.large_24} />
+                                    <Body>
+                                        Lorem ipsum dolor sit amet, consectetur
+                                        adipiscing elit, sed do eiusmod tempor
+                                        incididunt ut labore et dolore magna
+                                        aliqua.
+                                    </Body>
+                                    <Strut size={Spacing.large_24} />
+                                    <Button>Primary action</Button>
+                                </View>
+                            }
+                            closeButtonVisible={false}
```

Comment:
The original example in Netlify kept the default close buttons in the panels. It used the `mediaSize` from `MediaLayout` to determine when this prop should be true or false. I didn't do that here since we are moving away from `MediaLayout` and I decided it might be easier to remove the button entirely rather than figure out how to get this to work with CSS grid or adding a window resize listener something.